projects/angular/src/data/datagrid/built-in/filters/datagrid-numeric-filter.ts

```diff
@@ -140,6 +151,14 @@ export class DatagridNumericFilter<T = any>
     return this.minPlaceholder || this.commonStrings.keys.minValue;
   }
 
+  get minLabelValue() {
+    return this.maxPlaceholder || this.commonStrings.keys.fromLabel;
```

Comment:
```suggestion
  get fromLabelValue() {
    return this.fromLabel || this.commonStrings.keys.fromLabel;
```

I think my other bad naming suggestion caused confusion here. Sorry!